pkg/crdupdater/deployment.go

```diff
@@ -172,21 +171,21 @@ func (d *Deployment) patch(rc interface{}, isPatched bool) (bool, error) {
 				(!reflect.DeepEqual(
 					deploymentComparator{
 						Image:          oldContainer.Image,
-						Replicas:       d.oldDeployments[deployment.GetName()].Spec.Replicas,
-						MinCPU:         oldContainer.Resources.Requests.Cpu(),
-						MaxCPU:         oldContainer.Resources.Limits.Cpu(),
-						MinMem:         oldContainer.Resources.Requests.Memory(),
-						MaxMem:         oldContainer.Resources.Limits.Memory(),
+						Replicas:       *d.oldDeployments[deployment.GetName()].Spec.Replicas,
+						MinCPU:         oldContainer.Resources.Requests.Cpu().String(),
```

Comment:
This does make me question why pods did not restart before the memory change? 🤔  Is it because crdupdater only managed replicationControllers and not Deployments?